libs/ngx-dsfr/header/src/lib/header.component.scss

```diff
@@ -5,6 +5,84 @@
 
 // Actual styling
 .fr-header {
+  &.mini-mobile-header {
+    .fr-header__brand {
+      flex-wrap: nowrap;
+
+      .fr-header__brand-top {
+        .fr-header__logo {
+          padding-right: 0;
+          margin-right: -8px;
+
+          @media (min-width: 62em) {
+            padding-right: 1rem;
+            margin-right: 0.5rem;
+          }
+
+          .fr-logo {
+            &::before {
+              margin-bottom: -0.1rem;
+              margin-right: 1rem;
+
+              @media (min-width: 62em) {
+                margin-bottom: 0.25rem;
+                margin-right: unset;
+              }
+            }
+
+            &::after {
+              content: none;
+
+              @media (min-width: 62em) {
+                content: "";
+              }
+            }
+          }
+        }
+
+        .institution,
+        .fr-header__operator {
+          display: none;
+
+          @media (min-width: 62em) {
+            display: block;
+          }
+        }
+      }
+
+      .fr-header__service {
+        box-shadow: none;
+        margin-left: 0;
+        margin-right: 0;
+        width: auto;
+        padding: 1rem 0;
+
+        @media (min-width: 62em) {
+          padding: 1rem;
+        }
+
+        .fr-header__service-tagline {
+          display: none;
+
+          @media (min-width: 62em) {
+            display: block;
+          }
+        }
+      }
+    }
+  }
+
+  .mini-mobile-header__hidden {
```

Comment:
I forgot to tell you about that explicitly but, since we don't use these classes anymore in the Template, you can obviously remove this rule and the next...